code/game/objects/structures/janicart.dm

```diff
@@ -1,219 +1,3 @@
-//old style PIMP-CART
-/obj/structure/stool/bed/chair/janicart
-	name = "janicart"
-	desc = "A brave janitor cyborg gave its life to produce such an amazing combination of speed and utility."
-	icon = 'icons/obj/vehicles.dmi'
-	icon_state = "pussywagon"
-	anchored = 0
-	density = 1
-	var/obj/item/weapon/storage/bag/trash/mybag = null
-	var/callme = "pimpin' ride"	//how do people refer to it?
-	var/move_delay = 0
-	var/floorbuffer = 0
-	var/keytype = /obj/item/key/janitor
-
-/obj/structure/stool/bed/chair/janicart/New()
-	handle_rotation()
-
-/obj/structure/stool/bed/chair/janicart/Move(a, b, flag)
-	..()
-	if(floorbuffer)
-		var/turf/tile = loc
-		if(isturf(tile))
-			tile.clean_blood()
-			if (istype(tile, /turf/simulated/floor))
-				var/turf/simulated/floor/F = tile
-				F.dirt = 0
-			for(var/A in tile)
-				if(istype(A, /obj/effect))
-					if(is_cleanable(A))
-						qdel(A)
-
-/obj/structure/stool/bed/chair/janicart/examine(mob/user)
-	..()
-	if(floorbuffer)
-		to_chat(user, "It has been upgraded with a floor buffer.")
-
-
-/obj/structure/stool/bed/chair/janicart/attackby(obj/item/I, mob/user, params)
-	if(istype(I, keytype))
-		to_chat(user, "Hold [I] in one of your hands while you drive this [callme].")
-	else if(istype(I, /obj/item/weapon/storage/bag/trash))
-		if(keytype == /obj/item/key/janitor)
-			if(!user.drop_item())
-				return
-			to_chat(user, "<span class='notice'>You hook the trashbag onto the [callme].</span>")
-			I.loc = src
-			mybag = I
-	else if(istype(I, /obj/item/janiupgrade))
-		if(keytype == /obj/item/key/janitor)
-			floorbuffer = 1
-			qdel(I)
-			to_chat(user, "<span class='notice'>You upgrade the [callme] with the floor buffer.</span>")
-	update_icon()
-
-/obj/structure/stool/bed/chair/janicart/update_icon()
-	overlays.Cut()
-	if(mybag)
-		overlays += "cart_garbage"
-	if(floorbuffer)
-		overlays += "cart_buffer"
-
-/obj/structure/stool/bed/chair/janicart/attack_hand(mob/user)
-	if(mybag)
-		mybag.loc = get_turf(user)
-		user.put_in_hands(mybag)
-		mybag = null
-		update_icon()
-	else
-		..()
-
-
-/obj/structure/stool/bed/chair/janicart/relaymove(mob/user, direction)
-	if(user.stat || user.stunned || user.weakened || user.paralysis)
-		unbuckle_mob()
-	if(istype(user.l_hand, keytype) || istype(user.r_hand, keytype))
-		if(!Process_Spacemove(direction) || !has_gravity(src.loc) || move_delay || !isturf(loc))
-			return
-		step(src, direction)
-		update_mob()
-		handle_rotation()
-		if(istype(src.loc, /turf/simulated))

```

Comment:
This is probably the section you'll have to add if lube doesn't knock people off.